src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -181,6 +181,34 @@ def __init__(
         self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
         self.register_to_config(requires_safety_checker=requires_safety_checker)
 
+    def load_textual_inversion_embeddings(self, embeddings):
+        r"""
+        Loads textual inversion embeddings. Receives a dictionary with the following keys:
+        - `token`: name of the token to be added to the tokenizers' vocabulary
+        - `embedding`: path to the embedding of the token to be added to the text encoder's embedding matrix
+
+        Iters through the dictionary and adds the token to the tokenizer's vocabulary and the embedding to the text
+        encoder's embedding matrix.
+        """
+        for token, embedding_path in embeddings.items():
+            # check if token in tokenizer vocab
+            # if yes, raise exception
+            if token in self.tokenizer.get_vocab():
+                raise ValueError(
+                    f"Token {token} already in tokenizer vocabulary. Please choose a different token name."
+                )
+
+            # load embedding from embedding path then convert it to self.text_encoder's device and dtype
+            embedding = torch.load(embedding_path)
+            embedding = embedding.to(self.text_encoder.device)
+            embedding = embedding.to(self.text_encoder.dtype)
```

Comment:
Very cool!

Here, we should follow the standard embedding format of the textual inversion script. Which is a `dict`  like

```python
{"tok": embedding_vector}
```

That way, we can load all the embeddings available  in https://huggingface.co/sd-concepts-library

Reply:
Done, much better now.